heron/instance/src/java/com/twitter/heron/instance/HeronInstance.java

```diff
@@ -266,8 +275,13 @@ public static void main(String[] args) throws IOException {
     SingletonRegistry.INSTANCE.registerSingleton(SystemConfig.HERON_SYSTEM_CONFIG, systemConfig);
 
     // Create the protobuf Instance
-    PhysicalPlans.InstanceInfo instanceInfo = PhysicalPlans.InstanceInfo.newBuilder().
-        setTaskId(taskId).setComponentIndex(componentIndex).setComponentName(componentName).build();
+    PhysicalPlans.InstanceInfo.Builder instanceInfoBuilder
+        = PhysicalPlans.InstanceInfo.newBuilder().setTaskId(taskId)
+        .setComponentIndex(componentIndex).setComponentName(componentName);
+    if (remoteDebuggerPort != null) {
+      instanceInfoBuilder.setRemoteDebuggerPort(remoteDebuggerPort);
+    }
```

Comment:
Please add the port info in the logging message below if it is not there yet:

LOG.info("\nStarting instance " + instanceId + " for topology " + topologyName
         + " and topologyId " + topologyId + " for component " + componentName		         + " and topologyId " + topologyId + " for component " + componentName
         + " with taskId " + taskId + " and componentIndex " + componentIndex		         + " with taskId " + taskId + " and componentIndex " + componentIndex
         + " and stmgrId " + streamId + " and stmgrPort " + streamPort		         + " and stmgrId " + streamId + " and stmgrPort " + streamPort
         + " and metricsManagerPort " + metricsPort);		         + " and metricsManagerPort " + metricsPort);
 		 
     LOG.info("System Config: " + systemConfig);		     LOG.info("System Config: " + systemConfig);

Reply:
I have added this